scissors/makefile

```diff
@@ -0,0 +1,15 @@
+CC=gcc
+SOURCES=main.c
+EXECUTABLE=scissors
+OBJECTS=$(SOURCES:.c=.o)
+
+all: $(SOURCES) $(EXECUTABLE)
```

Comment:
`$(SOURCES)` is not necessary.
There are  $(EXECUTABLE): $(OBJECTS)` and `.c.o` -- it's enough to buld dependency tree.

Reply:
Fixed.